.github/workflows/test.yaml

```diff
@@ -27,3 +27,30 @@ jobs:
       - run: yarn run build
       - run: yarn run lint
       - run: yarn run test
+
+  zap_scan:
+    runs-on: ubuntu-latest
+    name: OWASP ZAP API Scan
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+      
+
+      - name: Install dependencies
+        uses: actions/setup-node@v3
+        with:
+          node-version: 18.12.1
+      # corepack is needed to run yarn through npm
+      - run: corepack enable
+      - run: yarn install --immutable
+      - run: yarn run build
+      - run: yarn run start:prod &
+      - run: sleep 3
+
+      - name: ZAP Scan
+        uses: zaproxy/action-api-scan@v0.3.1
+        with:
+          token: ${{ secrets.GITHUB_TOKEN }}
+          target: 'http://localhost:5000/openapi.json'
```

Comment:
can you explain this like how you link this target?

Reply:
https://github.com/Tonomy-Foundation/Tonomy-Communication/pull/54/files#diff-769911c416ccf8514d8fd941ae0abe8fb5c606ade0c218e22151a5f5f9f3d700R15

At this point we register the OpenAPI document to be served at this path